src/components/CohortBuilder/Summary/DemographicChart.js

```diff
@@ -4,19 +4,21 @@ import { withTheme } from 'emotion-theming';
 import { compose } from 'recompose';
 import { get, countBy, camelCase } from 'lodash';
 import Pie from 'chartkit/components/Pie';
-import { CardWrapper } from 'uikit/Card/styles';
 
-const CardSlotPies = styled(CardWrapper)`
-  height: 305px;
+const Container = styled('div')`
   display: flex;
   flex-wrap: wrap;
   flex-direction: row;
   justify-content: space-between;
-  padding: 10px 10px;
+  position: absolute;
+  left: 0px;
+  right: 0px;
+  top: 0px;
+  bottom: 0px;
 `;
 
 const DemographicChart = ({ data, theme }) => (
-  <CardSlotPies>
+  <Container>
```

Comment:
Since you named the container of the bar chart `BarChartContainer`, I suggest that you remain constant by naming this one `<PieChartContainer>`.